examples/hello.rs

```diff
@@ -16,10 +20,13 @@ fn main() {
         Ok(Response::new(Body::from(PHRASE)))
     }));
 
-    let server = Http::new()
-        .sleep_on_errors(true)
-        .bind(&addr, new_service)
-        .unwrap();
-    println!("Listening on http://{} with 1 thread.", server.local_addr().unwrap());
-    server.run().unwrap();
+    tokio::run(lazy(move || {
+        let server = Http::new()
+            .sleep_on_errors(true)
+            .bind(&addr, new_service)
+            .unwrap();
+
+        println!("Listening on http://{} with 1 thread.", server.local_addr().unwrap());
```

Comment:
All the lines printing about using 1 thread can probably be adjusted now :D

Reply:
Heh, will do!